cernopendata/config.py

```diff
@@ -284,7 +284,7 @@
             order=1,
         ),
         'mostrecent': dict(
-            fields=['recid'],
+            fields=['-created'],
```

Comment:
The sorting by `recid` was introduced on purpose, see #2418. This is because sometimes we add old content (e.g. second batch of Run2011 CMS open data) which will have "new" created timestamp, as it were, but "old" record IDs.
Hence I'll keep sorting by record IDs which approximately expresses sorting by "newer data".

Reply:
Okay, that makes sense. Why is record ID sorting an approximation and not an exact ordering?